src/clj/harja/kyselyt/tieverkko.sql

```diff
@@ -175,3 +175,37 @@ WHERE (:tiemax IS NULL OR tie < :tiemax::INT)
 GROUP BY tie
 ORDER BY tie
 LIMIT 100;
+
+-- name: onko-tr-alkuosa-olemassa?
+-- single?: true
+select exists(
+    SELECT "tr-numero"
+      FROM tr_osoitteet tr
+     WHERE tr."tr-numero" = :tie
+       AND tr."tr-osa" = :aosa
+       AND tr."tr-alkuetaisyys" <= :aet
+       AND tr."tr-loppuetaisyys" >= :aet);
+
+-- name: hae-tr-osan-tiedot
+SELECT "tr-numero", "tr-osa", MIN("tr-alkuetaisyys") as "tr-alkuetaisyys", MAX("tr-loppuetaisyys") as "tr-loppuetaisyys"
```

Comment:
Näitä selectejä voisi vähän rivittää jotta kysely selvemmin luettavissa

Reply:
Rivitetty ja "river" muoto korjattu